roles/openshift_control_plane/templates/master.yaml.v1.j2

```diff
@@ -185,7 +146,7 @@ oauthConfig:
   sessionConfig:
     sessionMaxAgeSeconds: {{ openshift.master.session_max_seconds }}
     sessionName: {{ openshift.master.session_name }}
-    sessionSecretsFile: {{ openshift.master.session_secrets_file }}
+    sessionSecretsFile: {{ openshift_master_session_secrets_file }}
```

Comment:
This change is required because openshift.master.session_secrets_file is empty by default, and not what we want.